java/src/org/openqa/selenium/support/events/EventFiringDecorator.java

```diff
@@ -157,7 +157,7 @@
  * but the decorator suppresses these exceptions), can't prevent execution of the decorated methods,
  * can't modify parameters and results of the methods.
  *
- * <p>Decorators that modify the behaviour of the underlying drivers should be implemented by
```

Comment:
Most of the comments use British English, not that it matters, but I am not sure if this is a substantial change that needs to be done.